clients/admin-ui/src/features/datastore-connections/add-connection/DatasetConfiguration.tsx

```diff
@@ -155,6 +162,11 @@ const DatasetConfiguration: React.FC = () => {
 
   return (
     <VStack alignItems="left">
+      {loadAllDatasetsError ? (
+        <Copy mb={4}>
+          There was a problem loading existing datasets, please try again.
```

Comment:
OK great improvements if there are errors loading bad datasets!

I didn't really notice the line of text though: `There was a problem loading existing datasets, please try again.` as it didn't stand out, it's not clear there was an error.  If you're going to add text directly to the page, could it be red, or perhaps a toast message instead?
<img width="1369" alt="Screen Shot 2023-01-12 at 11 07 40 AM" src="https://user-images.githubusercontent.com/9755598/212133194-2a6a872d-1941-4eb2-8330-addc249905f4.png">

Reply:
sure thing!

<img width="853" alt="image" src="https://user-images.githubusercontent.com/24641006/212142604-0f159c4c-5eb6-4bf7-99c5-f9d17a761cd8.png">